### ISSUE TITLE
One of the steps in Windows81-next-steps.md is to create an Apple developer account

### ISSUE THREAD

**Issue by phistuck** [manifoldjs/ManifoldJS#34]
I searched a little, but could not find the source of this file. Just so you know.

```
# Test - Build - Submit

## Test

1. Your application uses the same rendering and JavaScript Engine as the Internet Explorer Browser so most of you testing can be done on your website and in the browser.

2. To test your application on a device, download the ManifoldJS test harness from the Store, and follow the directions. _Coming soon!_

> **Note:** Looking for some debugging tools that work on all your platforms? Try [Vorlon.js](http://www.vorlonjs.io). It makes mobile testing a breeze, and works inside the app ManifoldJS apps.

## Build

1. Download and install Visual Studio (community edition works fine) and open the source code folder.

2. Open the Project in Visual Studio.

3. Use the store commands from Visual Studio to create the app package.

## Submit to Store
```
This is the bad part -
```
1. Register as an Apple developer [here](https://developer.apple.com/register/).

2. Reserve the name of your app.

3. Submit app package from store.
```

**Comment by boyofgreen** [comment]
Wow that's not good!  A copy and past error on my part I am sure.  I'll correct this.  Thanks for pointing out.

**Comment by nbellocam** [comment]
This is now fixed in [v0.1.3](https://github.com/manifoldjs/ManifoldJS/tree/v0.1.3) branch.
